ChangeLog

```diff
@@ -4,6 +4,10 @@ Pylint's ChangeLog
 What's New in Pylint 2.0?
 =========================
 
+	* Add a check `consider-swap-variables` for swapping variables with tuple unpacking
```

Comment:
I think we've decided to use spaces for the changelog. If you're using vim (like me), vim has a special filetype called 'changelog' which uses spaces by default. I had to put 

set expandtab

in `~/.vim/after/ftplugin/changelog.vim`

Reply:
Thank you for the hint :smile: 
=> fixed